src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/KeyValue.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
+
+data class KeyValue(
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏Could have used `Detail` for this?

Reply:
removed it as unused model.